doc/source/serve/http-guide.md

```diff
@@ -295,3 +296,15 @@ Here is a list of adapters; please feel free to [contribute more](https://github
     :members: json_to_ndarray, image_to_ndarray, starlette_request, json_request, pandas_read_json, json_to_multi_ndarray
 
 ```
+
+(serve-http-guide-keep-alive-timeout)=
+### Set keep alive timeout
+
+Serve uses a Uvicorn HTTP server internally to serve HTTP requests. By default, Uvicorn
+keeps HTTP connections alive for 5 seconds between requests. Modify the keep-alive
+timeout by setting the `keep_alive_timeout_s` in the `http_options` field of the Serve
+config files. This config is global to your Ray cluster, and it cannot be updated during
+runtime. You can also set `RAY_SERVE_HTTP_KEEP_ALIVE_TIMEOUT_S` environment variable to
+set the keep alive timeout. `RAY_SERVE_HTTP_KEEP_ALIVE_TIMEOUT_S` will take the
+precedence over the `keep_alive_timeout_s` config if both are set. You can find more
+information about Uvicorn's keep alive timeout [here](https://www.uvicorn.org/settings/#keep-alive-timeout).
```

Comment:
I think right now this is under `HTTP Adapters`:
<img width="257" alt="image" src="https://github.com/ray-project/ray/assets/15851518/122c5c60-b5db-4a4c-9976-c416a1d1fbd5">

Reply:
oops, thanks for catching it!